test/DotNetEnv.AspNetCore.Tests/Startup.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+using DotNetEnv;
+using DotNetEnv.AspNetCore;
+
+namespace DotNetEnv.AspNetCore.Tests
+{
+    public class Startup
+    {
+        // This method gets called by the runtime. Use this method to add services to the container.
+        // For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
+        public void ConfigureServices(IServiceCollection services)
+        {
+        }
+
+        // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
+        public void Configure(IApplicationBuilder app, IHostingEnvironment env)
+        {
+            if (env.IsDevelopment())
+            {
+                app.UseDeveloperExceptionPage();
+            }
+
+            app.UseDotNetEnv();

```

Comment:
To clarify: this line's purpose is simply to run `Env.Load();` right?

Replacing a one line call with just a different one line call begs the question of if this is the basis of future additions or solves some confusion.  Is the concern the lack of specificity/clarity around `Env.Load()`?  Or do you have grander plans for what comes next?

Could this be resolved by adding a section to the README explaining where the right place to call `Env.Load();` would be in an ASP.NET Core app?  (i.e. right here in `Startup.Configure`)